BlueprintUICommonControls/Sources/ScrollView.swift

```diff
@@ -329,6 +346,10 @@ fileprivate final class ScrollerWrapperView: UIView {
         if self.scrollView.contentInsetAdjustmentBehavior != scrollView.contentInsetAdjustmentBehavior {
             self.scrollView.contentInsetAdjustmentBehavior = scrollView.contentInsetAdjustmentBehavior
         }
+        
+        if let trigger = scrollView.setContentOffsetTrigger {
```

Comment:
I don't think you want this to be an `if let`, just a normal set; so if the value is cleared, it will be removed, right?

Reply:
- [ ] remove `if let` before setting trigger